packages/typespec-ts/test/modularUnit/scenarios/samples/parameters/parameterTypesCheck.md

```diff
@@ -11,6 +11,7 @@ model Foo {
 }
 
 model Widget {
+  unknownValue: unknown;
```

Comment:
@jiaodi add more cases with unknown type here:
for array, string, number, null and boolean.